src/main/java/com/kadai10/user/excepention/OccupationAlreadyExistsException.java

```diff
@@ -0,0 +1,18 @@
+package com.kadai10.user.excepention;
+
+/**
+ * ユーザーが既に存在する場合にスローされる例外クラスです. ユーザーが特定の条件を満たさない場合や、IDに対応するユーザーが存在する場合に使用されます。
+ */
+
+public class OccupationAlreadyExistsException extends RuntimeException {
```

Comment:
クラスに対するコメントとクラスの間に改行は不要かと思います。
```suggestion
 */
public class OccupationAlreadyExistsException extends RuntimeException {
```